lib/metrics.js

```diff
@@ -191,5 +202,6 @@ module.exports = {
     timestamp,
     openwhisk,
     flatten,
-    timeUntilTimeout
+    timeUntilTimeout,
+    DEFAULT_MAX_STRING_LENGTH
```

Comment:
Is this exported only for testing?

Reply:
Yeah, I wasn't sure if there was a better way to do it -- thoughts?

Reply:
`rewire` may be able to grab it, to verify.